presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/DockerContainer.java

```diff
@@ -155,26 +153,70 @@ private void copyFileToContainer(String containerPath, Runnable copy)
         log.info("Copied files into %s %s in %.1f s", this, containerPath, stopwatch.elapsed(MILLISECONDS) / 1000.);
     }
 
-    // Mounting a non-existing file results in docker creating a directory. This is often not the desired effect. Fail fast instead.
-    private static void verifyHostPath(String hostPath)
+    public void clearDependencies()
     {
-        if (!Files.exists(Paths.get(hostPath))) {
-            throw new IllegalArgumentException("Host path does not exist: " + hostPath);
+        dependencies.clear();
+    }
+
+    public void copyLogsToHostPath(Path hostPath)
+    {
```

Comment:
This should be in different commit

Reply:
Done.

Reply:
Well - I meant whole migration from `exposeLogsInHOstPath` to log copying. Currently it is part of `Add DockerContainer logical name`.